charts/graph-network-indexer/templates/indexer-tap-agent/servicemonitor.yaml

```
@@ -16,6 +16,7 @@ spec:
     matchLabels:
       {{- include "graph-network-indexer.selectorLabels" . | nindent 6 }}
       {{- $componentLabel | nindent 6 }}
+      serviceMonitorTarget: "true" # Additional label to prevent matching the headless service
```

yamllint

[warning] wrong indentation: expected 4 but found 6

(indentation)